genesyscloud/user_group_role_common.go

```diff
@@ -53,7 +53,7 @@ func getAssignedGrants(subjectID string, authAPI *platformclientv2.Authorization
 	return grants, resp, nil
 }
 
-func readSubjectRoles(d *schema.ResourceData, authAPI *platformclientv2.AuthorizationApi) (*schema.Set, *platformclientv2.APIResponse, diag.Diagnostics) {
+func ReadSubjectRoles(d *schema.ResourceData, authAPI *platformclientv2.AuthorizationApi) (*schema.Set, *platformclientv2.APIResponse, diag.Diagnostics) {
```

Comment:
move this file as well to its own package. most likely this wil be used by users and groups. and add a proxy for this.